synapseclient/client.py

```diff
@@ -2818,6 +2818,11 @@ def _downloadWikiAttachment(self, owner, wiki, filename, destination=None):
             destination = os.path.join(destination, filename)
         return self._downloadFile(url, destination)
 
+    def getWikiAttachments(self, wiki):
```

Comment:
I'm busy working on the SMC-Het docs, but need these attachment functions so I can sync the doc images to synapse. If you guys can help getting this into the API, it would be a great help.